stdlib/public/Darwin/Foundation/Foundation.swift

```diff
@@ -29,7 +29,152 @@ public let NSNotFound: Int = .max
+///     // International users will see a localization "1 + 1 = (1 + 1)".
+///     let localizedString = NSLocalizedString("string-interpolation",
+///                                             value: "1 + 1 = \(1 + 1)"
+///                                             comment: "A math equation.")
+///
+/// These strings must be string literal expressions. When you need
+/// to dynamically insert values within localized strings, set `value` to a
+/// format string, and use `String.localizedStringWithFormat(_:_:)` to insert
+/// those values.
+///
+///     // Translators will see "1 + 1 = %d" (they know what "%d" means).
+///     // International users will see a localization of "1 + 1 = 2".
+///     let format = NSLocalizedString("string-literal",
+///                                    value: "1 + 1 = %d",
+///                                    comment: "A math equation.")
+///     let localizedString = String.localizedStringWithFormat(format, (1 + 1))
+///
+/// Multi-line string literals are technically supported, but will result in
+/// unexpected behavior during internationalization. A newline will be inserted
+/// before and after the body of text within the string, and translators will
+/// likely preserve those in their internaliations.
+///
+/// To preserve some of the aesthetics of having newlines in the string mirrored
+/// in their code representation, string literal concatenation with the `+`
+/// operator can be used.
+///
+///     NSLocalizedString("multi-line-string-literal",
+///                       value: """
+///     This multi-line string literal won't work as expected.
+///     An extra newline added to the beginning and end of the string.
+///     """,
+///                       comment: "The description of a sample of code.")
+///
+///     NSLocalizedString("string-literal-contatenation",
+///                       value: "This string literal concatenated with"
+///                            + "this other string literal works just fine.",
+///                       comment: "The description of a sample of code.")
+///
+/// Luckily, since comments aren't localized, multi-line string literals can be
+/// safely used with `comment`.
+///
+/// Alternatives
+/// ------------
+///
+/// If having Xcode generate strings files from code isn't desired behavior,
+/// you should call `Bundle.localizedString(forKey:value:table:)` instead.
+///
+///     let greeting = Bundle.localizedString(forKey: "program-greeting",
+///                                           value: "Hello, World!",
+///                                           table: "Localization")
+///
+/// However, this requires the manual creation and management of that table's
+/// strings file.
+///
+///     /* Localization.strings */
+///
+///     /* A friendly greeting to the user when the program starts. */
+///     "program-greeting" = "Hello, World!";
+///
+/// **Note:** Although `NSLocalizedString(_:tableName:bundle:value:comment:)`
```

Comment:
The `Note:` syntax gets picked up when it's the first element of a bullet.
```suggestion
/// - Note: Although `NSLocalizedString(_:tableName:bundle:value:comment:)`
```